client/src/components/Header/Header.js

```diff
@@ -1,18 +1,35 @@
 import './Header.css';
 
+import { HEADERS } from '../../constants/header_constant';
+import PropTypes from 'prop-types';
 import React from 'react';
-import { SearchBox } from 'react-instantsearch-dom';
-import tardigrade from '../../images/tardigrade.png';
 
-const SEARCHBOX_CONFIG = {
-  placeholder: 'Search Ads...',
+const Header = props => {
+  const { currentPage, setCurrentPage } = props;
+
+  return (
+    <div className="header">
+      <h2 className="logo-text" onClick={() => setCurrentPage(HEADERS[0])}>
+        Tardigrade
+      </h2>
+      <div className="search-body">
+        {HEADERS.map((item, index) => (
+          <p
+            className={currentPage === item ? 'selected-header' : ''}
+            key={index}
+            onClick={() => setCurrentPage(item)}
```

Comment:
This looks like a page navigation bar, but you call it a header, which I think is a little confusing